### PR TITLE
add more tests to Xilinx FPGA testcase

### PR DESCRIPTION
**Pull request by hpourreza**
add few extra commands to test Xilinx FPGAs

### CODE REVIEW

File: Testscripts/Linux/fpga-card-validation.sh [hunk 1]
```diff
@@ -123,8 +123,35 @@ function validate_cards() {
         SetTestStateAborted
         return 1
     fi
+    echo "****************************" >> TestExecution.log
```
Comment (LiliDeng) [inline Testscripts/Linux/fpga-card-validation.sh]: LogMsg "****************************" will reach the same goal.

File: Testscripts/Linux/fpga-card-validation.sh [hunk 2]
```diff
@@ -123,8 +123,35 @@ function validate_cards() {
...
+    echo "Running xbutil scan" >> TestExecution.log
+    echo "****************************" >> TestExecution.log
     xbutil scan >> TestExecution.log
+
+    echo "****************************" >> TestExecution.log
+    echo "Running xbutil validate" >> TestExecution.log
+    echo "****************************" >> TestExecution.log
+    xbutil validate >> TestExecution.log
```
Comment (LiliDeng) [inline Testscripts/Linux/fpga-card-validation.sh]: xbutil validate >> TestExecution.log 2>&1
update redirect like this way, then error and info log will be written into TestExecution.log.

File: Testscripts/Linux/fpga-card-validation.sh [hunk 3]
```diff
@@ -135,7 +162,7 @@ function validate_cards() {
 # Source utils.sh
 . utils.sh || {
     echo "ERROR: unable to source utils.sh!"
-    echo "TestAborted" > state.txt
```
Comment (LiliDeng) [inline Testscripts/Linux/fpga-card-validation.sh]: When we have issues during source utils.sh, we can't use function SetTestStateAborted

File: Testscripts/Linux/fpga-card-validation.sh [hunk 4]
```diff
@@ -123,8 +123,35 @@ function validate_cards() {
         SetTestStateAborted
         return 1
     fi
+    echo "****************************" >> TestExecution.log
+    echo "Running xbutil scan" >> TestExecution.log
+    echo "****************************" >> TestExecution.log
     xbutil scan >> TestExecution.log
+
```
Comment (LiliDeng) [inline Testscripts/Linux/fpga-card-validation.sh]: should we have fail/pass check after each xbutil command?

File: Testscripts/Linux/fpga-card-validation.sh (hunk 1 above)
Comment (hpourreza) [reply Testscripts/Linux/fpga-card-validation.sh]: Done

File: Testscripts/Linux/fpga-card-validation.sh (hunk 2 above)
Comment (hpourreza) [reply Testscripts/Linux/fpga-card-validation.sh]: Done

File: Testscripts/Linux/fpga-card-validation.sh (hunk 4 above)
Comment (hpourreza) [reply Testscripts/Linux/fpga-card-validation.sh]: I added the check for return value `$?` after each `xbutil` run

File: Testscripts/Linux/fpga-card-validation.sh (hunk 3 above)
Comment (hpourreza) [reply Testscripts/Linux/fpga-card-validation.sh]: reverted back to what it was

**Comment by hpourreza** [comment]
@LiliDeng 
Is there any other change that I need to do?

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**